ssl-config-generator/index.html

```diff
@@ -39,12 +39,12 @@
     </script>
     <script>
         if (window.location.protocol != "https:")
-            window.location.href = "https:" + window.location.href.substring(window.location.protocol.length);
+            window.protocol = "https:";
```

Comment:
The ```if``` checks ```window.location.protocol```, but the new line modifies ```window.protocol```; maybe these should be the same variable?